docs/02-basic-tutorial.md

```diff
@@ -77,8 +77,6 @@ Pages (route handlers) in Reflex are React components. They all share the same `
 * `app-state` - the application state (more on that later)
 * `context` - the parsed URL with parameters (route segments or query string)
 
-To learn more about routing, read the [routing guide]().
```

Comment:
Ah, ok - thought it was an unwritten guide.

I'll leave the other changes alone - as you say needs a larger fix